### PR TITLE
[TRAFODION-1609]

### PR DESCRIPTION
**Pull request by mkby**
Change swstatus to give a simple "up" or "down" answer, instead of the current, confusing display

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
Looks good to me.

File: core/sqf/sql/scripts/install_local_hadoop [hunk 1]
```diff
@@ -847,20 +847,31 @@ EOF
...
-
+NUM_MYSQLD_PROCS=\`ps -aef | grep \$USER | grep mysqld | grep -v grep | wc -l\`
+if [[ \$NUM_MYSQLD_PROCS -ne 0 ]]; then
+    echo "\$NUM_MYSQLD_PROCS mysqld processes are running"
+    exit 0
+else
+    echo "ERROR: mysqld process is not running!"
+    exit 1
+fi
 EOF
```
Comment (selvaganesang) [inline core/sqf/sql/scripts/install_local_hadoop]: Is there a need to do it the same way for the other generated scripts too

Comment (mkby) [inline core/sqf/sql/scripts/install_local_hadoop]: What are the other generated scripts?

Comment (selvaganesang) [inline core/sqf/sql/scripts/install_local_hadoop]: See Line numbers 777 and above of this file. It has scripts like swstartall, swstopall, swstarthbase, swstophbase etc.

Comment (mkby) [inline core/sqf/sql/scripts/install_local_hadoop]: Thanks, I see. But they're all start/stop scripts, I don't want to modify them.

**Comment by traflm** [comment]
looks good, I think there is no need to change swstartall, swstopall, swstarthbase etc, this JIRA is to improve only status output to show better info.

**Comment by sureshsubbiah** [comment]
+1. If a change is necessary for swstart* / swstop* scripts, that could be a new JIRA.

**Comment by zellerh** [comment]
Looks good to me, too. I also don't think we need changes to other scripts. Thank you, this will make life easier for everyone!

**Comment by selvaganesang** [comment]
I was referring to the change to escape out the variables so that the script would contain the variables itself. My bad, I wasn't clear enough.

**Pull request merged**